spec/features/fill_in_whole_form_spec.rb

```diff
@@ -8,9 +9,30 @@
   let(:your_representatives_details_page) { ET3::Test::YourRepresentativesDetailsPage.new }
   let(:employers_contract_claim_page) { ET3::Test::EmployersContractClaimPage.new }
 
+  before do
+    stub_request(:post, "https://et-api-example.com/v2/repondents/response").
+      with(headers: { content_type: 'application/json', 'Accept': 'application/json' }).
+      to_return(
+        headers: { content_type: 'application/json' },
+        body:
+          {
+            "data": {
+              "reference": "992000000100",
+              "submitted_at": "2018-01-01 12:00:00 GMT+00",
+              "pdf": "s3/link/to/form/pdf"
+            }
+          }.to_json,
+        status: 201
+      )
+  end
+
   scenario "correctly will flow without error" do
 
-    respondents_details_page.load
+    start_page.load
+    expect(start_page).to be_displayed
+
+    start_page.next
+    expect(respondents_details_page).to be_displayed
```

Comment:
Ideally, your test examples should follow the 'Arrange', 'Act' and 'Assert' pattern - so no expect's in the 'arrange section'.  I suspect you are using these to wait for the page to be visible before you move on, but I doubt you need to.  If you really want to, then use wait_for_bla_bla_bla

Reply:
Fixed in c633430